install/docker/debian-docker-install.md

```diff
@@ -0,0 +1,122 @@
+# Install Docker for Kata Containers on Debian
+
+> **Note:**
+>
+> - This guide assumes you have
+>   [already installed the Kata Containers packages](../debian-installation-guide.md).
+> - this guide allows for installation with `systemd` or `sysVinit`init systems 
+> - notes are also provided for running multiple runtimes
+> - tested with `docker-ce=17.12.0~ce-0~debian`
+
+1. Install the latest version of Docker with the following commands:
+
+   > **Note:** This step is only required if Docker is not installed on the system.
+
+   ```bash
+   $ sudo apt-get install apt-transport-https ca-certificates curl gnupg2 software-properties-common  
+   $ curl -fsSL https://download.docker.com/linux/$(. /etc/os-release; echo "$ID")/gpg | sudo apt-key add -
+   $ sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/$(. /etc/os-release; echo "$ID") $(lsb_release -cs) stable"
```

Comment:
Hi @zeigerpuppy - The problem is just the `[arch=amd64]` as Kata is available for [multiple architectures](https://github.com/kata-containers/runtime/tree/master/arch). Hence I think we can just remove the `[arch=amd64]`?

Reply:
We can let `apt` worry about the available architectures the docker packages are available for I think:

- https://download.docker.com/linux/debian/dists/stretch/stable/

Reply:
done